biochemist-python/chapters/Linear_Regression.ipynb

```diff
@@ -0,0 +1,230 @@
+    "\n",
+    "However, some data can still be analyzed by simple linear regression. Perhaps the most common case is the protein assay. Whether you use Lowry, Bradford or BCA methods, it is still most common to use a linear regression fit to the results.\n",
+    "\n",
+    "In this module, we will explore linear regression in Jupyter notebooks using Python. Please keep in mind - this is just a beginning. If you take a course in data science, you are likely to encounter a much deeper look at linear regression where you explore the relationships among many variables in a dataset.\n",
+    "\n",
+    "We're going to build on the previous lesson, using the pandas library to import the data for this lesson. Then, we will perform the linear regression using the scipy library. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Libraries you will need\n",
+    "\n",
+    "In previous lessons, we have used `os`, `numpy`, and `pandas`. In this lesson, we will add the SciPy library. We will use the dot notation we introduced earlier to access the functions in these libraries. I have expanded our table of libraries to help you keep track of your tools. Please note that the abbreviations listed are just the most common. You can actually define any abbreviation you like, but it is best to use the conventional abbreviations. This will help future coders (including yourself six months from now) as they work with the code. \n",
+    "\n",
+    "| Library | Uses | Abbreviation |\n",
+    "| :------- | :----: | :------------: |\n",
+    "| os | file management in operating systems | os |\n",
+    "| numpy | calculations | np  | \n",
+    "| pandas | data management | pd |\n",
+    "| scipy | calculations and statistics | sc or sp | \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Stages of this module\n",
+    "1. Importing the correct libraries\n",
+    "1. Importing data with pandas\n",
+    "1. Running simple linear regression to print out the desired statistics\n",
+    "\n",
+    "The first part of the module is modeled after an exercise in Charlie Weiss's excellent online textbook, *Scientific Computing for Chemists*, which you can find on his GitHub site, [SciCompforChemists](https://github.com/weisscharlesj/SciCompforChemists)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the libraries you need\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import scipy as sp"
```

Comment:
Need more of an explanation of scipy.

Reply:
I expanded the explanation of SciPy based on a nice source on the web.

Reply:
I can't see where I can respond to your comments on table formatting, so I'll add that comment here. In the Jupyter Book they appear to be double spaced. I would like them to be single spaced and shaded a bit, the way they appear in the Jupyter notebook. It's not urgent, but it would be nice.